academy/4-my-own-chain/cosmjs-gui.md

```diff
@@ -481,19 +481,19 @@ Looking into `GameContainer`, you see that `componentDidMount` gets all the game
     }
     ```
 
-    Note how:
+    Note:
 
-    * You force `isSaved` to `true` since it is indeed always saved in the blockchain.
-    * You do not care about saving `game.index` to state because it is actually passed in `IGameContainerProps.index`.
+    * You force `isSaved` to `true` since it is always saved in the blockchain.
+    * Saving `game.index` to state is unimportant because it is actually passed in `IGameContainerProps.index`.
     * By having a separate `loadGame`, you can call it again if you know the game has changed.
 
-Restart `npm start` and you should now see your game. If you have access to `checkersd`, or any other way to send a transaction, you can make a move from the command line and refresh to confirm the change. If not, wait for the part about playing.
+Restart `npm start` and you should now see your game. If you have access to `checkersd`, or any other way to send a transaction, you can make a move from the command line and refresh to confirm the change. If not, wait for the procedure about playing.
 
 ![Game with one move played](/show-game-1-move.png)
 
 ## Integrate with Keplr
 
-So you have _only_ made it possible to show the state of games and of the blockchain. That is good. You should let your users poke around without unnecessarily asking them to connect. However, to create a game, or play in one, you need to make transactions. This is where you need first to make the integration with Keplr possible.
+So far you have _only_ made it possible to show the state of games and of the blockchain. This allows your users poke around without unnecessarily asking them to connect. However, to create a game or play in one, you need to make transactions. This is where you need to make integration with Keplr possible.
```

Comment:
```suggestion
So far you have _only_ made it possible to show the state of games and of the blockchain. This allows your users to poke around without unnecessarily asking them to connect. However, to create a game or play in one, you need to make transactions. This is where you need to make integration with Keplr possible.
```
Not sure it is missing "to" or not. @AndrewB9lab

Reply:
It certainly is, thank you for the catch!